### ISSUE TITLE
0.8.2: Unable to scroll tableview neither to top nor button nor to specific row

### ISSUE THREAD

**Issue by ser1us** [moredip/Frank#54]
Frank version (gem):  0.8.2

I have the following step definition

When /^I scroll details to bottom$/ do
  tables_scrolled = frankly_map( "tableView", "scrollToBottom", "tag" ) 
  puts tables_scrolled
end

The problem is, nothing happens, it just tells me it was successful even tough I can't see 

Anyone figured out how to to do this?

Thanks!

**Comment by moredip** [comment]
Looks like you found a bug, which should be fixed in the 0.8.3 gem I just pushed. Can you upgrade the frank server code to 0.8.3 and try again? If you are still having issue please let me know.

**NOTE:**
to upgrade the server, you'll need to do a `gem update frank-cucumber`, followed by running `frank-skeleton update-server` in your project directory to update the frank server static library itself.

**Comment by ser1us** [comment]
Hi 

I updated my local gem to 0.8.3:

> Christophs-MacBook-Pro:Frank christoph$ gem list --local | grep "frank-cucumber"
> frank-cucumber (0.8.3)

And then I executed `frank-skeleton update-server` as told, but my git repository thinks nothing has changed (which is a bit strange right, at least the "libFrank.a" should change I guess?

Anyway, I ran the cucumber test again, but still no scrolling tables.

I think the weirdest part here is that git doesn't see any changes on the frank stuff, even tough I did run the update-server command.

Cheers,
Christoph

Update: Just in case I deleted the frank_static_resource.bundle and the libFrank.a and ran `frank-skeleton update-server` again (still no changes in git). Didn't work out, but at least I tried also this option :)
